tests/Validation/ValidationBase64Test.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace Illuminate\Tests\Validation;
+
+use Illuminate\Support\Str;
+use Illuminate\Validation\Rules\Base64;
+use PHPUnit\Framework\TestCase;
+
+class ValidationBase64Test extends TestCase
+{
+    public function testBase64StringPassValidation()
+    {
+        $string = base64_encode(Str::random());
```

Comment:
It doesn't _really_ matter, but there's no reason not to use hardcoded strings instead of `Str::random()` in your test cases.

`$string = base64_encode('Hello world');`

Reply:
Is replaced :-)=